person.rb

```diff
@@ -1,15 +1,21 @@
 require './nameable'
-
+require './rental'
 class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
```

Comment:
- [ ] I would suggest you please use `Getters and Setters for @rentals` for example:
``attr_accessor :name, :age, :rental``
However, if you foresee a situation where you might need to access or modify @rental from outside the Person class in the future, you can add attr_accessor :rental to make it accessible externally. Please implement it